src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -2723,6 +2747,31 @@ internal BoundExpression GenerateConversionForAssignment(TypeSymbol targetType,
             Debug.Assert(expression.HasAnyErrors && expression.Kind != BoundKind.UnboundLambda, "Missing a case in implicit conversion error reporting");
         }
 
+        private void GenerateImplicitConversionErrorsForTupleLiteralArguments(
+            DiagnosticBag diagnostics, 
+            CSharpSyntaxNode syntax,
+            ImmutableArray<BoundExpression> tupleArguments, 
+            ImmutableArray<TypeSymbol> targetElementTypes)
+        {
+            var argLength = tupleArguments.Length;
+
+            // report all leaf elements of the tuple literal that failed to convert
+            // NOTE: we are not responsible for reporting use site errors here, just the failed leaf conversions.
```

Comment:
Could you clarify why we are ignoring use-site errors. Is it because they were already reported elsewhere?